pset-functions.js

```diff
@@ -91,6 +104,24 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount(phrase){
+  const cleanedPhrase = phrase.replace(/[,.]/g, '')
+  const splitWords = cleanedPhrase.split(' ')
+  const wordCount = {}
+
+  splitWords.forEach((splitWord) => {
+    if(wordCount[splitWord] === undefined) {
+      wordCount[splitWord] = 1
+    } else {
+      wordCount[splitWord] += 1
+      }
+    })
+    return wordCount
+
+}
+
+console.log(wordCount("Baby shark, doo doo doo doo doo doo"))
```

Comment:
@nmikhol Nice work here creating a function that checks the splits the string into an array and loop over those elements to either set to 1 if the word doesn't exist or increment by 1 if it does.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L103)